src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars--4 {
+  display: flex;
```

Comment:
At the top should be the basic BEM bloc selector - `stars`. In `stars--4` you should add styles only to display 4 active stars